cloudify_agent/installer/runners/winrm_runner.py

```diff
@@ -161,7 +161,8 @@ def _chk(res):
                     output=res.std_out)
                 if raise_on_failure:
                     raise error
-                self.logger.error(error)
+                self.logger.exception("WinRM command ended with an error",
```

Comment:
Printing an `Exception` object without some sort of a prefix is a bad practice IMO